src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
+---
+```
+
+## Introduction section
+
+After you insert the front matter, insert the introduction, which typically just includes text and an image. If you need help with inserting images, see the example at the bottom.
+
+## Tiles
+
+Tiles are a series of boxes that contain the main subject areas for your product. You should just list these in order you want them to appear, and the cascading style sheet will render them across the page.
+
+Here's an example of a tile: 
+
+```
+<LandingPageTileGrid>
+  <LandingPageTile
+    title="INSERT_YOUR_BOX_TITLE_HERE."
+    icon="fe-INSERT_THE_ICON_NAME"
+  >
+
+Insert tile content here...
+
+</LandingPageTile>
+```
+For each tile, do the following:
+
+* Insert a value for `title` that explains the purpose of the category.
+* Insert a value for `icon` by prefixing one of the [Feather Icons](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) with `fe-` so it is recognized (for example, fe-alert-triangle).
```

Comment:
We have quite a few other icons available in the docs site that aren't available in the theme (we should probably change that). The theme is pretty limited in the bundled icons. The full set of icons available are both the icons documented in the theme and the ones listed here: https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js

This website is also handy to show the available feather icons: https://feathericons.com/.  We don't have all of them pulled in, but new icons can always be requested.

Reply:
@jerelmiller, I dropped the ball on this and would like to follow up. Can you check out the callout tip I added that lists the various options for finding icons? Still, do if an icon isn't listed in the theme, do users need to request it before they can use it?

Reply:
Just checked out the doc and that looks perfect! Thanks for updating!

Reply:
> do users need to request it before they can use it?

Not necessarily. If someone wants to add additional feather icons, we'll take contributions for them! They will show up in a PR review, so we should be good there.

Reply:
@jerelmiller, one of our technical writers wants to use the "database" Feather icon, which isn't in the theme. How would she go about getting it so it's available with "fe-database"?

Reply:
Its just a little bit involved, but this is the process I follow:

1. Go to feathericons.com
2. Download the "database" feather icon by clicking on the icon itself.
3. Once downloaded, open the SVG in your text editor
4. Grab the "guts" of the SVG, which is everything in between the `svg` tags. For example, if the `svg` looks like this: `<svg><path m="1"></path></svg>`, then you'll want to grab only the `<path m="1"></path>` portion. 
5. Open the list of feather icons at [`src/@newrelic/gatsby-theme-newrelic/icons/feather.js`](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js)
6. Add an entry for `database` and assign the code from step 4 to it. This particular icon has multiple paths, so you'll want that `<>` wrapper around it like you'll see with other icons
7. Save that `feather.js` file.

The `fe-` prefix gets automatically added. Once that icon is added, you can use it with the `Icon` component with `<Icon name="fe-database" />`.

I'd also be happy to add this myself if needed.

Reply:
This this is more complicated that I realized, I am going to create a section about this in "Embed images" and just insert a link to that doc from here.

Reply:
I think its totally reasonable just to say "ask the engineering team" or "open an issue with an icon request". The above steps are for those who want to understand how to do it themselves.